// @strapi/admin/build/2077.31a2d91e.chunk.js
(self.webpackChunk_strapi_admin=self.webpackChunk_strapi_admin||[]).push([[2077],{74201:(y,C,h)=>{"use strict";y.exports=h(45214)},45214:function(y,C,h){(function(x,v){y.exports=v(h(32735),h(19615),h(103))})(this,function(x,v,m){return function(t){var f={};function e(o){if(f[o])return f[o].exports;var i=f[o]={i:o,l:!1,exports:{}};return t[o].call(i.exports,i,i.exports,e),i.l=!0,i.exports}return e.m=t,e.c=f,e.d=function(o,i,p){e.o(o,i)||Object.defineProperty(o,i,{enumerable:!0,get:p})},e.r=function(o){typeof Symbol!="undefined"&&Symbol.toStringTag&&Object.defineProperty(o,Symbol.toStringTag,{value:"Module"}),Object.defineProperty(o,"__esModule",{value:!0})},e.t=function(o,i){if(1&i&&(o=e(o)),8&i||4&i&&typeof o=="object"&&o&&o.__esModule)return o;var p=Object.create(null);if(e.r(p),Object.defineProperty(p,"default",{enumerable:!0,value:o}),2&i&&typeof o!="string")for(var u in o)e.d(p,u,function(d){return o[d]}.bind(null,u));return p},e.n=function(o){var i=o&&o.__esModule?function(){return o.default}:function(){return o};return e.d(i,"a",i),i},e.o=function(o,i){return Object.prototype.hasOwnProperty.call(o,i)},e.p="",e(e.s=64)}([function(t,f,e){t.exports=e(19)()},function(t,f){t.exports=x},function(t,f){t.exports=v},function(t,f){t.exports=function(e,o){return o||(o=e.slice(0)),Object.freeze(Object.defineProperties(e,{raw:{value:Object.freeze(o)}}))},t.exports.default=t.exports,t.exports.__esModule=!0},function(t,f,e){var o=e(24);t.exports=function(i,p){if(i==null)return{};var u,d,c=o(i,p);if(Object.getOwnPropertySymbols){var g=Object.getOwnPropertySymbols(i);for(d=0;d<g.length;d++)u=g[d],p.indexOf(u)>=0||Object.prototype.propertyIsEnumerable.call(i,u)&&(c[u]=i[u])}return c},t.exports.default=t.exports,t.exports.__esModule=!0},function(t,f){function e(){return t.exports=e=Object.assign||function(o){for(var i=1;i<arguments.length;i++){var p=arguments[i];for(var u in p)Object.prototype.hasOwnProperty.call(p,u)&&(o[u]=p[u])}return o},t.exports.default=t.exports,t.exports.__esModule=!0,e.apply(this,arguments)}t.exports=e,t.exports.default=t.exports,t.exports.__esModule=!0},function(t,f,e){"use strict";e.r(f),e.d(f,"Box",function(){return j});var o,i=e(3),p=e.n(i),u=e(2),d=e.n(u),c=e(7),g=e(1),O=e.n(g),S=e(0),a=e.n(S),l=function(n){return O.a.createElement("div",n)},w={background:void 0,borderColor:void 0,color:void 0,hiddenS:!1,hiddenXS:!1,padding:void 0,paddingTop:void 0,paddingRight:void 0,paddingBottom:void 0,paddingLeft:void 0,hasRadius:!1,shadow:void 0,children:null,shrink:void 0,grow:void 0,basis:void 0,flex:void 0,_hover:function(){}},b={_hover:a.a.func,background:a.a.string,basis:a.a.oneOfType([a.a.string,a.a.string]),borderColor:a.a.string,children:a.a.oneOfType([a.a.node,a.a.string]),color:a.a.string,flex:a.a.oneOfType([a.a.string,a.a.string]),grow:a.a.oneOfType([a.a.string,a.a.string]),hasRadius:a.a.bool,hiddenS:a.a.bool,hiddenXS:a.a.bool,padding:a.a.oneOfType([a.a.number,a.a.arrayOf(a.a.number)]),paddingBottom:a.a.oneOfType([a.a.number,a.a.arrayOf(a.a.number)]),paddingLeft:a.a.oneOfType([a.a.number,a.a.arrayOf(a.a.number)]),paddingRight:a.a.oneOfType([a.a.number,a.a.arrayOf(a.a.number)]),paddingTop:a.a.oneOfType([a.a.number,a.a.arrayOf(a.a.number)]),shadow:a.a.string,shrink:a.a.oneOfType([a.a.string,a.a.string])};l.defaultProps=w,l.propTypes=b;var T={color:!0},j=d.a.div.withConfig({shouldForwardProp:function(n,r){return!T[n]&&r(n)}})(o||(o=p()([`
  // Font
  font-size: `,`;

  // Colors
  background: `,`;
  color: `,`;

  // Spaces
  `,`
  `,`
  `,`
  `,`
  `,`
  `,`
  `,`
  `,`
  `,`

  // Responsive hiding
  `,`
  `,`
  

  // Borders
  border-radius: `,`;
  border-style: `,`;
  border-width: `,`;
  border-color: `,`;
  border: `,`;

  // Shadows
  box-shadow: `,`;

  // Handlers
  pointer-events: `,`;
  &:hover {
    `,`
  }

  // Display
  display: `,`;

  // Position
  position: `,`;
  left: `,`;
  right: `,`;
  top: `,`;
  bottom: `,`;
  z-index: `,`;
  overflow: `,`;
  cursor: `,`;

  // Size
  width: `,`;
  max-width: `,`;
  min-width: `,`;
  height: `,`;
  max-height: `,`;
  min-height: `,`;

  // Animation
  transition: `,`;
  transform: `,`;
  animation: `,`;

  //Flexbox children props
  flex-shrink: `,`;
  flex-grow: `,`;
  flex-basis: `,`;
  flex: `,`;

  // Text
  text-align: `,`;
  text-transform: `,`;
  line-height: `,`;

  // Cursor
  cursor: `,`;
`])),function(n){var r=n.fontSize;return n.theme.fontSizes[r]||r},function(n){var r=n.theme,s=n.background;return r.colors[s]},function(n){var r=n.theme,s=n.color;return r.colors[s]},function(n){var r=n.theme,s=n.padding;return Object(c.a)("padding",s,r)},function(n){var r=n.theme,s=n.paddingTop;return Object(c.a)("padding-top",s,r)},function(n){var r=n.theme,s=n.paddingRight;return Object(c.a)("padding-right",s,r)},function(n){var r=n.theme,s=n.paddingBottom;return Object(c.a)("padding-bottom",s,r)},function(n){var r=n.theme,s=n.paddingLeft;return Object(c.a)("padding-left",s,r)},function(n){var r=n.theme,s=n.marginLeft;return Object(c.a)("margin-left",s,r)},function(n){var r=n.theme,s=n.marginRight;return Object(c.a)("margin-right",s,r)},function(n){var r=n.theme,s=n.marginTop;return Object(c.a)("margin-top",s,r)},function(n){var r=n.theme,s=n.marginBottom;return Object(c.a)("margin-bottom",s,r)},function(n){var r=n.theme;return n.hiddenS?"".concat(r.mediaQueries.tablet," { display: none; }"):void 0},function(n){var r=n.theme;return n.hiddenXS?"".concat(r.mediaQueries.mobile," { display: none; }"):void 0},function(n){var r=n.theme,s=n.hasRadius,k=n.borderRadius;return s?r.borderRadius:k},function(n){return n.borderStyle},function(n){return n.borderWidth},function(n){var r=n.borderColor;return n.theme.colors[r]},function(n){var r=n.theme,s=n.borderColor,k=n.borderStyle,M=n.borderWidth;if(s&&!k&&!M)return"1px solid ".concat(r.colors[s])},function(n){var r=n.theme,s=n.shadow;return r.shadows[s]},function(n){return n.pointerEvents},function(n){var r=n._hover,s=n.theme;return r?r(s):void 0},function(n){return n.display},function(n){return n.position},function(n){var r=n.left;return n.theme.spaces[r]||r},function(n){var r=n.right;return n.theme.spaces[r]||r},function(n){var r=n.top;return n.theme.spaces[r]||r},function(n){var r=n.bottom;return n.theme.spaces[r]||r},function(n){return n.zIndex},function(n){return n.overflow},function(n){return n.cursor},function(n){var r=n.width;return n.theme.spaces[r]||r},function(n){var r=n.maxWidth;return n.theme.spaces[r]||r},function(n){var r=n.minWidth;return n.theme.spaces[r]||r},function(n){var r=n.height;return n.theme.spaces[r]||r},function(n){var r=n.maxHeight;return n.theme.spaces[r]||r},function(n){var r=n.minHeight;return n.theme.spaces[r]||r},function(n){return n.transition},function(n){return n.transform},function(n){return n.animation},function(n){return n.shrink},function(n){return n.grow},function(n){return n.basis},function(n){return n.flex},function(n){return n.textAlign},function(n){return n.textTransform},function(n){return n.lineHeight},function(n){return n.cursor});j.defaultProps=w,j.propTypes=b},function(t,f,e){"use strict";var o=e(10),i=e.n(o),p=e(13),u=e.n(p);f.a=function(d,c,g){var O=c;if(Array.isArray(c)||u()(c)!=="object"||(O=[c==null?void 0:c.desktop,c==null?void 0:c.tablet,c==null?void 0:c.mobile]),O!==void 0){if(Array.isArray(O)){var S=O,a=i()(S,3),l=a[0],w=a[1],b=a[2],T="".concat(d,": ").concat(g.spaces[l],";");return w!==void 0&&(T+="".concat(g.mediaQueries.tablet,`{
          `).concat(d,": ").concat(g.spaces[w],`;
        }`)),b!==void 0&&(T+="".concat(g.mediaQueries.mobile,`{
          `).concat(d,": ").concat(g.spaces[b],`;
        }`)),T}var j=g.spaces[O]||O;return"".concat(d,": ").concat(j,";")}}},function(t,f,e){"use strict";e.r(f),e.d(f,"Typography",function(){return j});var o,i=e(3),p=e.n(i),u=e(2),d=e.n(u),c=["alpha","beta","delta","epsilon","omega","pi","sigma"],g=e(1),O=e.n(g),S=e(0),a=e.n(S),l=function(n){return O.a.createElement("div",n)},w={ellipsis:!1,fontWeight:void 0,fontSize:void 0,lineHeight:void 0,textColor:void 0,textTransform:void 0,variant:"omega"},b={ellipsis:a.a.bool,fontSize:a.a.oneOfType([a.a.number,a.a.string]),fontWeight:a.a.string,lineHeight:a.a.oneOfType([a.a.number,a.a.string]),textColor:a.a.string,textTransform:a.a.string,variant:a.a.oneOf(c)};l.defaultProps=w,l.propTypes=b;var T={fontSize:!0,fontWeight:!0},j=d.a.span.withConfig({shouldForwardProp:function(n,r){return!T[n]&&r(n)}})(o||(o=p()([`
  font-weight: `,`;
  font-size: `,`;
  line-height: `,`;
  color: `,`;
  text-transform: `,`;
  `,`
  `,`
`])),function(n){var r=n.theme,s=n.fontWeight;return r.fontWeights[s]},function(n){var r=n.theme,s=n.fontSize;return r.fontSizes[s]},function(n){var r=n.theme,s=n.lineHeight;return r.lineHeights[s]},function(n){var r=n.theme,s=n.textColor;return r.colors[s||"neutral800"]},function(n){return n.textTransform},function(n){return n.ellipsis&&`
    display: block;
    white-space: nowrap;
    overflow: hidden;
    text-overflow: ellipsis;
  `},function(n){var r=n.variant,s=n.theme;switch(r){case"alpha":return`
        font-weight: `.concat(s.fontWeights.bold,`;
        font-size: `).concat(s.fontSizes[5],`;
        line-height: `).concat(s.lineHeights[2],`;
      `);case"beta":return`
        font-weight: `.concat(s.fontWeights.bold,`;
        font-size: `).concat(s.fontSizes[4],`;
        line-height: `).concat(s.lineHeights[1],`;
      `);case"delta":return`
        font-weight: `.concat(s.fontWeights.semiBold,`;
        font-size: `).concat(s.fontSizes[3],`;
        line-height: `).concat(s.lineHeights[2],`;
      `);case"epsilon":return`
        font-size: `.concat(s.fontSizes[3],`;
        line-height: `).concat(s.lineHeights[6],`;
      `);case"omega":return`
        font-size: `.concat(s.fontSizes[2],`;
        line-height: `).concat(s.lineHeights[4],`;
      `);case"pi":return`
        font-size: `.concat(s.fontSizes[1],`;
        line-height: `).concat(s.lineHeights[3],`;
      `);case"sigma":return`
        font-weight: `.concat(s.fontWeights.bold,`;
        font-size: `).concat(s.fontSizes[0],`;
        line-height: `).concat(s.lineHeights[5],`;
        text-transform: uppercase;
      `);default:return`
        font-size: `.concat(s.fontSizes[2],`;
      `)}});j.defaultProps=w,j.propTypes=b},function(t,f,e){"use strict";e.r(f),e.d(f,"Flex",function(){return n});var o,i=e(3),p=e.n(i),u=e(2),d=e.n(u),c=e(6),g=e(7),O=e(1),S=e.n(O),a=e(0),l=e.n(a),w=function(r){return S.a.createElement("div",r)},b={alignItems:"center",basis:void 0,direction:"row",gap:void 0,inline:!1,justifyContent:void 0,reverse:!1,wrap:void 0},T={alignItems:l.a.string,basis:l.a.oneOfType([l.a.string,l.a.number]),direction:l.a.string,gap:l.a.oneOfType([l.a.shape({desktop:l.a.number,mobile:l.a.number,tablet:l.a.number}),l.a.number,l.a.arrayOf(l.a.number),l.a.string]),inline:l.a.bool,justifyContent:l.a.string,reverse:l.a.bool,shrink:l.a.number,wrap:l.a.string};w.defaultProps=b,w.propTypes=T;var j={direction:!0},n=d()(c.Box).withConfig({shouldForwardProp:function(r,s){return!j[r]&&s(r)}})(o||(o=p()([`
  align-items: `,`;
  display: `,`;
  flex-direction: `,`;
  flex-shrink: `,`;
  flex-wrap: `,`;
  `,`};
  justify-content: `,`;
`])),function(r){return r.alignItems},function(r){return r.inline?"inline-flex":"flex"},function(r){return r.direction},function(r){return r.shrink},function(r){return r.wrap},function(r){var s=r.gap,k=r.theme;return Object(g.a)("gap",s,k)},function(r){return r.justifyContent});n.defaultProps=b,n.propTypes=T},function(t,f,e){var o=e(25),i=e(26),p=e(22),u=e(27);t.exports=function(d,c){return o(d)||i(d,c)||p(d,c)||u()},t.exports.default=t.exports,t.exports.__esModule=!0},,,function(t,f){function e(o){return typeof Symbol=="function"&&typeof Symbol.iterator=="symbol"?(t.exports=e=function(i){return typeof i},t.exports.default=t.exports,t.exports.__esModule=!0):(t.exports=e=function(i){return i&&typeof Symbol=="function"&&i.constructor===Symbol&&i!==Symbol.prototype?"symbol":typeof i},t.exports.default=t.exports,t.exports.__esModule=!0),e(o)}t.exports=e,t.exports.default=t.exports,t.exports.__esModule=!0},,,function(t,f,e){"use strict";e.d(f,"b",function(){return o}),e.d(f,"c",function(){return i}),e.d(f,"a",function(){return p});var o=function(u){return function(d){var c=d.theme,g=d.size;return c.sizes[u][g]}},i=function(){var u=arguments.length>0&&arguments[0]!==void 0?arguments[0]:"&";return function(d){var c=d.theme,g=d.hasError;return`
      outline: none;
      box-shadow: 0;
      transition-property: border-color, box-shadow, fill;
      transition-duration: 0.2s;

      `.concat(u,`:focus-within {
        border: 1px solid `).concat(g?c.colors.danger600:c.colors.primary600,`;
        box-shadow: `).concat(g?c.colors.danger600:c.colors.primary600,` 0px 0px 0px 2px;
      }
    `)}},p=function(u){var d=u.theme;return`
  position: relative;
  outline: none;
  
  &:after {
    transition-property: all;
    transition-duration: 0.2s;
    border-radius: 8px;
    content: '';
    position: absolute;
    top: -4px;
    bottom: -4px;
    left: -4px;
    right: -4px;
    border: 2px solid transparent;
  }

  &:focus-visible {
    outline: none;
    &:after {
      border-radius: 8px;
      content: '';
      position: absolute;
      top: -5px;
      bottom: -5px;
      left: -5px;
      right: -5px;
      border: 2px solid `.concat(d.colors.primary600,`;
    }
  }
`)}},,,function(t,f,e){"use strict";var o=e(20);function i(){}function p(){}p.resetWarningCache=i,t.exports=function(){function u(g,O,S,a,l,w){if(w!==o){var b=new Error("Calling PropTypes validators directly is not supported by the `prop-types` package. Use PropTypes.checkPropTypes() to call them. Read more at http://fb.me/use-check-prop-types");throw b.name="Invariant Violation",b}}function d(){return u}u.isRequired=u;var c={array:u,bool:u,func:u,number:u,object:u,string:u,symbol:u,any:u,arrayOf:d,element:u,elementType:u,instanceOf:d,node:u,objectOf:d,oneOf:d,oneOfType:d,shape:d,exact:d,checkPropTypes:p,resetWarningCache:i};return c.PropTypes=c,c}},function(t,f,e){"use strict";t.exports="SECRET_DO_NOT_PASS_THIS_OR_YOU_WILL_BE_FIRED"},function(t,f){t.exports=function(e,o){(o==null||o>e.length)&&(o=e.length);for(var i=0,p=new Array(o);i<o;i++)p[i]=e[i];return p},t.exports.default=t.exports,t.exports.__esModule=!0},function(t,f,e){var o=e(21);t.exports=function(i,p){if(i){if(typeof i=="string")return o(i,p);var u=Object.prototype.toString.call(i).slice(8,-1);return u==="Object"&&i.constructor&&(u=i.constructor.name),u==="Map"||u==="Set"?Array.from(i):u==="Arguments"||/^(?:Ui|I)nt(?:8|16|32)(?:Clamped)?Array$/.test(u)?o(i,p):void 0}},t.exports.default=t.exports,t.exports.__esModule=!0},,function(t,f){t.exports=function(e,o){if(e==null)return{};var i,p,u={},d=Object.keys(e);for(p=0;p<d.length;p++)i=d[p],o.indexOf(i)>=0||(u[i]=e[i]);return u},t.exports.default=t.exports,t.exports.__esModule=!0},function(t,f){t.exports=function(e){if(Array.isArray(e))return e},t.exports.default=t.exports,t.exports.__esModule=!0},function(t,f){t.exports=function(e,o){var i=e==null?null:typeof Symbol!="undefined"&&e[Symbol.iterator]||e["@@iterator"];if(i!=null){var p,u,d=[],c=!0,g=!1;try{for(i=i.call(e);!(c=(p=i.next()).done)&&(d.push(p.value),!o||d.length!==o);c=!0);}catch(O){g=!0,u=O}finally{try{c||i.return==null||i.return()}finally{if(g)throw u}}return d}},t.exports.default=t.exports,t.exports.__esModule=!0},function(t,f){t.exports=function(){throw new TypeError(`Invalid attempt to destructure non-iterable instance.
In order to be iterable, non-array objects must have a [Symbol.iterator]() method.`)},t.exports.default=t.exports,t.exports.__esModule=!0},,,,,,,,,,,,,,,,,,,,,,,,function(t,f){t.exports=m},,,,,,,,,,,,,function(t,f,e){"use strict";e.r(f),e.d(f,"TextButton",function(){return P});var o,i,p,u=e(5),d=e.n(u),c=e(4),g=e.n(c),O=e(3),S=e.n(O),a=e(1),l=e.n(a),w=e(0),b=e.n(w),T=e(2),j=e.n(T),n=e(51),r=e.n(n),s=e(6),k=e(8),M=e(9),_=e(16),F=["children","startIcon","endIcon","onClick","disabled","loading"],L=Object(T.keyframes)(o||(o=S()([`
  from {
    transform: rotate(0deg);
  }
  to {
    transform: rotate(359deg);
  }
`]))),Q=j.a.div(i||(i=S()([`
  animation: `,` 2s infinite linear;
  will-change: transform;
`])),L),U=j()(M.Flex)(p||(p=S()([`
  background: transparent;
  border: none;

  &[aria-disabled='true'] {
    pointer-events: none;
    svg path {
      fill: `,`;
    }
  }

  svg {
    display: flex;
    font-size: `,`rem;
  }

  svg path {
    fill: `,`;
  }

  `,`
`])),function(z){return z.theme.colors.neutral600},.625,function(z){return z.theme.colors.primary600},_.a),P=l.a.forwardRef(function(z,X){var I=z.children,B=z.startIcon,A=z.endIcon,H=z.onClick,W=z.disabled,E=z.loading,G=g()(z,F),V=H&&!W?H:void 0,R=W||E;return l.a.createElement(U,d()({ref:X,"aria-disabled":R,onClick:V,as:"button",type:"button"},G),(B||E)&&l.a.createElement(s.Box,{as:"span",paddingRight:2,"aria-hidden":!0},E?l.a.createElement(Q,null,l.a.createElement(r.a,null)):B),l.a.createElement(k.Typography,{variant:"pi",textColor:R?"neutral600":"primary600"},I),A&&l.a.createElement(s.Box,{as:"span",paddingLeft:2,"aria-hidden":!0},A))});P.displayName="TextButton",P.defaultProps={disabled:!1,startIcon:void 0,endIcon:void 0,loading:!1,onClick:void 0},P.propTypes={children:b.a.node.isRequired,disabled:b.a.bool,endIcon:b.a.element,loading:b.a.bool,onClick:b.a.func,startIcon:b.a.element}}])})},56671:(y,C,h)=>{var x=h(35897),v=h(74741),m=v(x);y.exports=m},52920:(y,C,h)=>{var x=h(56671),v=h(81580);function m(t,f){var e=-1,o=v(t)?Array(t.length):[];return x(t,function(i,p,u){o[++e]=f(i,p,u)}),o}y.exports=m},50690:(y,C,h)=>{var x=h(26460),v=h(80166),m=h(88904),t=h(52920),f=h(36024),e=h(76535),o=h(1397),i=h(15362),p=h(77236);function u(d,c,g){c.length?c=x(c,function(a){return p(a)?function(l){return v(l,a.length===1?a[0]:a)}:a}):c=[i];var O=-1;c=x(c,e(m));var S=t(d,function(a,l,w){var b=x(c,function(T){return T(a)});return{criteria:b,index:++O,value:a}});return f(S,function(a,l){return o(a,l,g)})}y.exports=u},36024:y=>{function C(h,x){var v=h.length;for(h.sort(x);v--;)h[v]=h[v].value;return h}y.exports=C},40709:(y,C,h)=>{var x=h(22945);function v(m,t){if(m!==t){var f=m!==void 0,e=m===null,o=m===m,i=x(m),p=t!==void 0,u=t===null,d=t===t,c=x(t);if(!u&&!c&&!i&&m>t||i&&p&&d&&!u&&!c||e&&p&&d||!f&&d||!o)return 1;if(!e&&!i&&!c&&m<t||c&&f&&o&&!e&&!i||u&&f&&o||!p&&o||!d)return-1}return 0}y.exports=v},1397:(y,C,h)=>{var x=h(40709);function v(m,t,f){for(var e=-1,o=m.criteria,i=t.criteria,p=o.length,u=f.length;++e<p;){var d=x(o[e],i[e]);if(d){if(e>=u)return d;var c=f[e];return d*(c=="desc"?-1:1)}}return m.index-t.index}y.exports=v},74741:(y,C,h)=>{var x=h(81580);function v(m,t){return function(f,e){if(f==null)return f;if(!x(f))return m(f,e);for(var o=f.length,i=t?o:-1,p=Object(f);(t?i--:++i<o)&&e(p[i],i,p)!==!1;);return f}}y.exports=v},98978:(y,C,h)=>{var x=h(40314),v=h(50690),m=h(94569),t=h(63108),f=m(function(e,o){if(e==null)return[];var i=o.length;return i>1&&t(e,o[0],o[1])?o=[]:i>2&&t(o[0],o[1],o[2])&&(o=[o[0]]),v(e,x(o,1),[])});y.exports=f}}]);
