// @strapi/admin/build/4982.c2a311b7.chunk.js
(self.webpackChunk_strapi_admin=self.webpackChunk_strapi_admin||[]).push([[4982],{12001:(Q,ue,k)=>{"use strict";Q.exports=k(27901)},27901:function(Q,ue,k){var X=k(98399);(function(I,R){Q.exports=R(k(32735),k(19615),k(60672),k(103))})(this,function(I,R,E,v){return function(r){var s={};function e(o){if(s[o])return s[o].exports;var a=s[o]={i:o,l:!1,exports:{}};return r[o].call(a.exports,a,a.exports,e),a.l=!0,a.exports}return e.m=r,e.c=s,e.d=function(o,a,p){e.o(o,a)||Object.defineProperty(o,a,{enumerable:!0,get:p})},e.r=function(o){typeof Symbol!="undefined"&&Symbol.toStringTag&&Object.defineProperty(o,Symbol.toStringTag,{value:"Module"}),Object.defineProperty(o,"__esModule",{value:!0})},e.t=function(o,a){if(1&a&&(o=e(o)),8&a||4&a&&typeof o=="object"&&o&&o.__esModule)return o;var p=Object.create(null);if(e.r(p),Object.defineProperty(p,"default",{enumerable:!0,value:o}),2&a&&typeof o!="string")for(var l in o)e.d(p,l,function(d){return o[d]}.bind(null,l));return p},e.n=function(o){var a=o&&o.__esModule?function(){return o.default}:function(){return o};return e.d(a,"a",a),a},e.o=function(o,a){return Object.prototype.hasOwnProperty.call(o,a)},e.p="",e(e.s=99)}([function(r,s,e){r.exports=e(19)()},function(r,s){r.exports=I},function(r,s){r.exports=R},function(r,s){r.exports=function(e,o){return o||(o=e.slice(0)),Object.freeze(Object.defineProperties(e,{raw:{value:Object.freeze(o)}}))},r.exports.default=r.exports,r.exports.__esModule=!0},function(r,s,e){var o=e(24);r.exports=function(a,p){if(a==null)return{};var l,d,f=o(a,p);if(Object.getOwnPropertySymbols){var h=Object.getOwnPropertySymbols(a);for(d=0;d<h.length;d++)l=h[d],p.indexOf(l)>=0||Object.prototype.propertyIsEnumerable.call(a,l)&&(f[l]=a[l])}return f},r.exports.default=r.exports,r.exports.__esModule=!0},function(r,s){function e(){return r.exports=e=Object.assign||function(o){for(var a=1;a<arguments.length;a++){var p=arguments[a];for(var l in p)Object.prototype.hasOwnProperty.call(p,l)&&(o[l]=p[l])}return o},r.exports.default=r.exports,r.exports.__esModule=!0,e.apply(this,arguments)}r.exports=e,r.exports.default=r.exports,r.exports.__esModule=!0},function(r,s,e){"use strict";e.r(s),e.d(s,"Box",function(){return O});var o,a=e(3),p=e.n(a),l=e(2),d=e.n(l),f=e(7),h=e(1),m=e.n(h),S=e(0),i=e.n(S),g=function(n){return m.a.createElement("div",n)},y={background:void 0,borderColor:void 0,color:void 0,hiddenS:!1,hiddenXS:!1,padding:void 0,paddingTop:void 0,paddingRight:void 0,paddingBottom:void 0,paddingLeft:void 0,hasRadius:!1,shadow:void 0,children:null,shrink:void 0,grow:void 0,basis:void 0,flex:void 0,_hover:function(){}},x={_hover:i.a.func,background:i.a.string,basis:i.a.oneOfType([i.a.string,i.a.string]),borderColor:i.a.string,children:i.a.oneOfType([i.a.node,i.a.string]),color:i.a.string,flex:i.a.oneOfType([i.a.string,i.a.string]),grow:i.a.oneOfType([i.a.string,i.a.string]),hasRadius:i.a.bool,hiddenS:i.a.bool,hiddenXS:i.a.bool,padding:i.a.oneOfType([i.a.number,i.a.arrayOf(i.a.number)]),paddingBottom:i.a.oneOfType([i.a.number,i.a.arrayOf(i.a.number)]),paddingLeft:i.a.oneOfType([i.a.number,i.a.arrayOf(i.a.number)]),paddingRight:i.a.oneOfType([i.a.number,i.a.arrayOf(i.a.number)]),paddingTop:i.a.oneOfType([i.a.number,i.a.arrayOf(i.a.number)]),shadow:i.a.string,shrink:i.a.oneOfType([i.a.string,i.a.string])};g.defaultProps=y,g.propTypes=x;var w={color:!0},O=d.a.div.withConfig({shouldForwardProp:function(n,t){return!w[n]&&t(n)}})(o||(o=p()([`
  // Font
  font-size: `,`;

  // Colors
  background: `,`;
  color: `,`;

  // Spaces
  `,`
  `,`
  `,`
  `,`
  `,`
  `,`
  `,`
  `,`
  `,`

  // Responsive hiding
  `,`
  `,`
  

  // Borders
  border-radius: `,`;
  border-style: `,`;
  border-width: `,`;
  border-color: `,`;
  border: `,`;

  // Shadows
  box-shadow: `,`;

  // Handlers
  pointer-events: `,`;
  &:hover {
    `,`
  }

  // Display
  display: `,`;

  // Position
  position: `,`;
  left: `,`;
  right: `,`;
  top: `,`;
  bottom: `,`;
  z-index: `,`;
  overflow: `,`;
  cursor: `,`;

  // Size
  width: `,`;
  max-width: `,`;
  min-width: `,`;
  height: `,`;
  max-height: `,`;
  min-height: `,`;

  // Animation
  transition: `,`;
  transform: `,`;
  animation: `,`;

  //Flexbox children props
  flex-shrink: `,`;
  flex-grow: `,`;
  flex-basis: `,`;
  flex: `,`;

  // Text
  text-align: `,`;
  text-transform: `,`;
  line-height: `,`;

  // Cursor
  cursor: `,`;
`])),function(n){var t=n.fontSize;return n.theme.fontSizes[t]||t},function(n){var t=n.theme,c=n.background;return t.colors[c]},function(n){var t=n.theme,c=n.color;return t.colors[c]},function(n){var t=n.theme,c=n.padding;return Object(f.a)("padding",c,t)},function(n){var t=n.theme,c=n.paddingTop;return Object(f.a)("padding-top",c,t)},function(n){var t=n.theme,c=n.paddingRight;return Object(f.a)("padding-right",c,t)},function(n){var t=n.theme,c=n.paddingBottom;return Object(f.a)("padding-bottom",c,t)},function(n){var t=n.theme,c=n.paddingLeft;return Object(f.a)("padding-left",c,t)},function(n){var t=n.theme,c=n.marginLeft;return Object(f.a)("margin-left",c,t)},function(n){var t=n.theme,c=n.marginRight;return Object(f.a)("margin-right",c,t)},function(n){var t=n.theme,c=n.marginTop;return Object(f.a)("margin-top",c,t)},function(n){var t=n.theme,c=n.marginBottom;return Object(f.a)("margin-bottom",c,t)},function(n){var t=n.theme;return n.hiddenS?"".concat(t.mediaQueries.tablet," { display: none; }"):void 0},function(n){var t=n.theme;return n.hiddenXS?"".concat(t.mediaQueries.mobile," { display: none; }"):void 0},function(n){var t=n.theme,c=n.hasRadius,j=n.borderRadius;return c?t.borderRadius:j},function(n){return n.borderStyle},function(n){return n.borderWidth},function(n){var t=n.borderColor;return n.theme.colors[t]},function(n){var t=n.theme,c=n.borderColor,j=n.borderStyle,_=n.borderWidth;if(c&&!j&&!_)return"1px solid ".concat(t.colors[c])},function(n){var t=n.theme,c=n.shadow;return t.shadows[c]},function(n){return n.pointerEvents},function(n){var t=n._hover,c=n.theme;return t?t(c):void 0},function(n){return n.display},function(n){return n.position},function(n){var t=n.left;return n.theme.spaces[t]||t},function(n){var t=n.right;return n.theme.spaces[t]||t},function(n){var t=n.top;return n.theme.spaces[t]||t},function(n){var t=n.bottom;return n.theme.spaces[t]||t},function(n){return n.zIndex},function(n){return n.overflow},function(n){return n.cursor},function(n){var t=n.width;return n.theme.spaces[t]||t},function(n){var t=n.maxWidth;return n.theme.spaces[t]||t},function(n){var t=n.minWidth;return n.theme.spaces[t]||t},function(n){var t=n.height;return n.theme.spaces[t]||t},function(n){var t=n.maxHeight;return n.theme.spaces[t]||t},function(n){var t=n.minHeight;return n.theme.spaces[t]||t},function(n){return n.transition},function(n){return n.transform},function(n){return n.animation},function(n){return n.shrink},function(n){return n.grow},function(n){return n.basis},function(n){return n.flex},function(n){return n.textAlign},function(n){return n.textTransform},function(n){return n.lineHeight},function(n){return n.cursor});O.defaultProps=y,O.propTypes=x},function(r,s,e){"use strict";var o=e(10),a=e.n(o),p=e(13),l=e.n(p);s.a=function(d,f,h){var m=f;if(Array.isArray(f)||l()(f)!=="object"||(m=[f==null?void 0:f.desktop,f==null?void 0:f.tablet,f==null?void 0:f.mobile]),m!==void 0){if(Array.isArray(m)){var S=m,i=a()(S,3),g=i[0],y=i[1],x=i[2],w="".concat(d,": ").concat(h.spaces[g],";");return y!==void 0&&(w+="".concat(h.mediaQueries.tablet,`{
          `).concat(d,": ").concat(h.spaces[y],`;
        }`)),x!==void 0&&(w+="".concat(h.mediaQueries.mobile,`{
          `).concat(d,": ").concat(h.spaces[x],`;
        }`)),w}var O=h.spaces[m]||m;return"".concat(d,": ").concat(O,";")}}},function(r,s,e){"use strict";e.r(s),e.d(s,"Typography",function(){return O});var o,a=e(3),p=e.n(a),l=e(2),d=e.n(l),f=["alpha","beta","delta","epsilon","omega","pi","sigma"],h=e(1),m=e.n(h),S=e(0),i=e.n(S),g=function(n){return m.a.createElement("div",n)},y={ellipsis:!1,fontWeight:void 0,fontSize:void 0,lineHeight:void 0,textColor:void 0,textTransform:void 0,variant:"omega"},x={ellipsis:i.a.bool,fontSize:i.a.oneOfType([i.a.number,i.a.string]),fontWeight:i.a.string,lineHeight:i.a.oneOfType([i.a.number,i.a.string]),textColor:i.a.string,textTransform:i.a.string,variant:i.a.oneOf(f)};g.defaultProps=y,g.propTypes=x;var w={fontSize:!0,fontWeight:!0},O=d.a.span.withConfig({shouldForwardProp:function(n,t){return!w[n]&&t(n)}})(o||(o=p()([`
  font-weight: `,`;
  font-size: `,`;
  line-height: `,`;
  color: `,`;
  text-transform: `,`;
  `,`
  `,`
`])),function(n){var t=n.theme,c=n.fontWeight;return t.fontWeights[c]},function(n){var t=n.theme,c=n.fontSize;return t.fontSizes[c]},function(n){var t=n.theme,c=n.lineHeight;return t.lineHeights[c]},function(n){var t=n.theme,c=n.textColor;return t.colors[c||"neutral800"]},function(n){return n.textTransform},function(n){return n.ellipsis&&`
    display: block;
    white-space: nowrap;
    overflow: hidden;
    text-overflow: ellipsis;
  `},function(n){var t=n.variant,c=n.theme;switch(t){case"alpha":return`
        font-weight: `.concat(c.fontWeights.bold,`;
        font-size: `).concat(c.fontSizes[5],`;
        line-height: `).concat(c.lineHeights[2],`;
      `);case"beta":return`
        font-weight: `.concat(c.fontWeights.bold,`;
        font-size: `).concat(c.fontSizes[4],`;
        line-height: `).concat(c.lineHeights[1],`;
      `);case"delta":return`
        font-weight: `.concat(c.fontWeights.semiBold,`;
        font-size: `).concat(c.fontSizes[3],`;
        line-height: `).concat(c.lineHeights[2],`;
      `);case"epsilon":return`
        font-size: `.concat(c.fontSizes[3],`;
        line-height: `).concat(c.lineHeights[6],`;
      `);case"omega":return`
        font-size: `.concat(c.fontSizes[2],`;
        line-height: `).concat(c.lineHeights[4],`;
      `);case"pi":return`
        font-size: `.concat(c.fontSizes[1],`;
        line-height: `).concat(c.lineHeights[3],`;
      `);case"sigma":return`
        font-weight: `.concat(c.fontWeights.bold,`;
        font-size: `).concat(c.fontSizes[0],`;
        line-height: `).concat(c.lineHeights[5],`;
        text-transform: uppercase;
      `);default:return`
        font-size: `.concat(c.fontSizes[2],`;
      `)}});O.defaultProps=y,O.propTypes=x},function(r,s,e){"use strict";e.r(s),e.d(s,"Flex",function(){return n});var o,a=e(3),p=e.n(a),l=e(2),d=e.n(l),f=e(6),h=e(7),m=e(1),S=e.n(m),i=e(0),g=e.n(i),y=function(t){return S.a.createElement("div",t)},x={alignItems:"center",basis:void 0,direction:"row",gap:void 0,inline:!1,justifyContent:void 0,reverse:!1,wrap:void 0},w={alignItems:g.a.string,basis:g.a.oneOfType([g.a.string,g.a.number]),direction:g.a.string,gap:g.a.oneOfType([g.a.shape({desktop:g.a.number,mobile:g.a.number,tablet:g.a.number}),g.a.number,g.a.arrayOf(g.a.number),g.a.string]),inline:g.a.bool,justifyContent:g.a.string,reverse:g.a.bool,shrink:g.a.number,wrap:g.a.string};y.defaultProps=x,y.propTypes=w;var O={direction:!0},n=d()(f.Box).withConfig({shouldForwardProp:function(t,c){return!O[t]&&c(t)}})(o||(o=p()([`
  align-items: `,`;
  display: `,`;
  flex-direction: `,`;
  flex-shrink: `,`;
  flex-wrap: `,`;
  `,`};
  justify-content: `,`;
`])),function(t){return t.alignItems},function(t){return t.inline?"inline-flex":"flex"},function(t){return t.direction},function(t){return t.shrink},function(t){return t.wrap},function(t){var c=t.gap,j=t.theme;return Object(h.a)("gap",c,j)},function(t){return t.justifyContent});n.defaultProps=x,n.propTypes=w},function(r,s,e){var o=e(25),a=e(26),p=e(22),l=e(27);r.exports=function(d,f){return o(d)||a(d,f)||p(d,f)||l()},r.exports.default=r.exports,r.exports.__esModule=!0},function(r,s,e){"use strict";e.d(s,"a",function(){return o});var o={DOWN:"ArrowDown",UP:"ArrowUp",RIGHT:"ArrowRight",LEFT:"ArrowLeft",ESCAPE:"Escape",ENTER:"Enter",SPACE:" ",TAB:"Tab",END:"End",HOME:"Home",DELETE:"Delete",PAGE_UP:"PageUp",PAGE_DOWN:"PageDown",BACKSPACE:"Backspace",CLEAR:"Clear"}},,function(r,s){function e(o){return typeof Symbol=="function"&&typeof Symbol.iterator=="symbol"?(r.exports=e=function(a){return typeof a},r.exports.default=r.exports,r.exports.__esModule=!0):(r.exports=e=function(a){return a&&typeof Symbol=="function"&&a.constructor===Symbol&&a!==Symbol.prototype?"symbol":typeof a},r.exports.default=r.exports,r.exports.__esModule=!0),e(o)}r.exports=e,r.exports.default=r.exports,r.exports.__esModule=!0},function(r,s,e){"use strict";e.d(s,"a",function(){return p});var o=e(1),a=0,p=function(l,d){return Object(o.useRef)(d||"".concat(l,"-").concat(++a)).current}},,function(r,s,e){"use strict";e.d(s,"b",function(){return o}),e.d(s,"c",function(){return a}),e.d(s,"a",function(){return p});var o=function(l){return function(d){var f=d.theme,h=d.size;return f.sizes[l][h]}},a=function(){var l=arguments.length>0&&arguments[0]!==void 0?arguments[0]:"&";return function(d){var f=d.theme,h=d.hasError;return`
      outline: none;
      box-shadow: 0;
      transition-property: border-color, box-shadow, fill;
      transition-duration: 0.2s;

      `.concat(l,`:focus-within {
        border: 1px solid `).concat(h?f.colors.danger600:f.colors.primary600,`;
        box-shadow: `).concat(h?f.colors.danger600:f.colors.primary600,` 0px 0px 0px 2px;
      }
    `)}},p=function(l){var d=l.theme;return`
  position: relative;
  outline: none;
  
  &:after {
    transition-property: all;
    transition-duration: 0.2s;
    border-radius: 8px;
    content: '';
    position: absolute;
    top: -4px;
    bottom: -4px;
    left: -4px;
    right: -4px;
    border: 2px solid transparent;
  }

  &:focus-visible {
    outline: none;
    &:after {
      border-radius: 8px;
      content: '';
      position: absolute;
      top: -5px;
      bottom: -5px;
      left: -5px;
      right: -5px;
      border: 2px solid `.concat(d.colors.primary600,`;
    }
  }
`)}},,,function(r,s,e){"use strict";var o=e(20);function a(){}function p(){}p.resetWarningCache=a,r.exports=function(){function l(h,m,S,i,g,y){if(y!==o){var x=new Error("Calling PropTypes validators directly is not supported by the `prop-types` package. Use PropTypes.checkPropTypes() to call them. Read more at http://fb.me/use-check-prop-types");throw x.name="Invariant Violation",x}}function d(){return l}l.isRequired=l;var f={array:l,bool:l,func:l,number:l,object:l,string:l,symbol:l,any:l,arrayOf:d,element:l,elementType:l,instanceOf:d,node:l,objectOf:d,oneOf:d,oneOfType:d,shape:d,exact:d,checkPropTypes:p,resetWarningCache:a};return f.PropTypes=f,f}},function(r,s,e){"use strict";r.exports="SECRET_DO_NOT_PASS_THIS_OR_YOU_WILL_BE_FIRED"},function(r,s){r.exports=function(e,o){(o==null||o>e.length)&&(o=e.length);for(var a=0,p=new Array(o);a<o;a++)p[a]=e[a];return p},r.exports.default=r.exports,r.exports.__esModule=!0},function(r,s,e){var o=e(21);r.exports=function(a,p){if(a){if(typeof a=="string")return o(a,p);var l=Object.prototype.toString.call(a).slice(8,-1);return l==="Object"&&a.constructor&&(l=a.constructor.name),l==="Map"||l==="Set"?Array.from(a):l==="Arguments"||/^(?:Ui|I)nt(?:8|16|32)(?:Clamped)?Array$/.test(l)?o(a,p):void 0}},r.exports.default=r.exports,r.exports.__esModule=!0},function(r,s,e){"use strict";e.r(s),e.d(s,"Stack",function(){return _});var o,a,p=e(5),l=e.n(p),d=e(4),f=e.n(d),h=e(3),m=e.n(h),S=e(1),i=e.n(S),g=e(0),y=e.n(g),x=e(2),w=e.n(x),O=e(9),n=["horizontal","spacing","size"],t={size:!0},c=w()(O.Flex).withConfig({shouldForwardProp:function(b,C){return!t[b]&&C(b)}})(o||(o=m()([`
  & > * {
    margin-top: 0;
    margin-bottom: 0;
  }

  & > * + * {
    margin-top: `,`;
  }
`])),function(b){var C=b.theme,M=b.spacing;return C.spaces[M]}),j=w()(O.Flex).withConfig({shouldForwardProp:function(b,C){return!t[b]&&C(b)}})(a||(a=m()([`
  & > * {
    margin-left: 0;
    margin-right: 0;
  }

  & > * + * {
    margin-left: `,`;
  }
`])),function(b){var C=b.theme,M=b.spacing;return C.spaces[M]}),_=Object(S.forwardRef)(function(b,C){var M=b.horizontal,U=b.spacing,F=b.size,A=f()(b,n);return F&&X.warn('Deprecation warning: Usage of "size" prop in Stack component is deprecated. This is discouraged and will be removed in the next major release. Please use "spacing" instead'),M?i.a.createElement(j,l()({ref:C,spacing:U||F},A)):i.a.createElement(c,l()({direction:"column",alignItems:"stretch",ref:C,spacing:U||F},A))});_.displayName="Stack",_.defaultProps={horizontal:!1,size:void 0,spacing:void 0},_.propTypes={horizontal:y.a.bool,size:y.a.number,spacing:y.a.number}},function(r,s){r.exports=function(e,o){if(e==null)return{};var a,p,l={},d=Object.keys(e);for(p=0;p<d.length;p++)a=d[p],o.indexOf(a)>=0||(l[a]=e[a]);return l},r.exports.default=r.exports,r.exports.__esModule=!0},function(r,s){r.exports=function(e){if(Array.isArray(e))return e},r.exports.default=r.exports,r.exports.__esModule=!0},function(r,s){r.exports=function(e,o){var a=e==null?null:typeof Symbol!="undefined"&&e[Symbol.iterator]||e["@@iterator"];if(a!=null){var p,l,d=[],f=!0,h=!1;try{for(a=a.call(e);!(f=(p=a.next()).done)&&(d.push(p.value),!o||d.length!==o);f=!0);}catch(m){h=!0,l=m}finally{try{f||a.return==null||a.return()}finally{if(h)throw l}}return d}},r.exports.default=r.exports,r.exports.__esModule=!0},function(r,s){r.exports=function(){throw new TypeError(`Invalid attempt to destructure non-iterable instance.
In order to be iterable, non-array objects must have a [Symbol.iterator]() method.`)},r.exports.default=r.exports,r.exports.__esModule=!0},function(r,s){r.exports=function(e,o,a){return o in e?Object.defineProperty(e,o,{value:a,enumerable:!0,configurable:!0,writable:!0}):e[o]=a,e},r.exports.default=r.exports,r.exports.__esModule=!0},,function(r,s,e){var o=e(42),a=e(43),p=e(22),l=e(44);r.exports=function(d){return o(d)||a(d)||p(d)||l()},r.exports.default=r.exports,r.exports.__esModule=!0},,,,function(r,s){r.exports=E},,,,,,,function(r,s,e){"use strict";e.r(s),e.d(s,"Icon",function(){return x});var o,a=e(5),p=e.n(a),l=e(3),d=e.n(l),f=e(1),h=e.n(f),m=e(0),S=e.n(m),i=e(6),g=e(2),y=e.n(g)()(i.Box)(o||(o=d()([`
  path {
    fill: `,`;
  }
  `,`
`])),function(w){var O=w.color;return w.theme.colors[O]},function(w){var O=w.theme;return(0,w.colors)(O)}),x=h.a.forwardRef(function(w,O){return h.a.createElement(y,p()({ref:O},w))});x.displayName="Icon",x.defaultProps={color:"neutral600",colors:function(){}},x.propTypes={color:S.a.string,colors:S.a.func}},function(r,s,e){var o=e(21);r.exports=function(a){if(Array.isArray(a))return o(a)},r.exports.default=r.exports,r.exports.__esModule=!0},function(r,s){r.exports=function(e){if(typeof Symbol!="undefined"&&e[Symbol.iterator]!=null||e["@@iterator"]!=null)return Array.from(e)},r.exports.default=r.exports,r.exports.__esModule=!0},function(r,s){r.exports=function(){throw new TypeError(`Invalid attempt to spread non-iterable instance.
In order to be iterable, non-array objects must have a [Symbol.iterator]() method.`)},r.exports.default=r.exports,r.exports.__esModule=!0},,,,,,,function(r,s){r.exports=v},,,,,,,,,,,,,function(r,s,e){"use strict";e.r(s),e.d(s,"TextButton",function(){return A});var o,a,p,l=e(5),d=e.n(l),f=e(4),h=e.n(f),m=e(3),S=e.n(m),i=e(1),g=e.n(i),y=e(0),x=e.n(y),w=e(2),O=e.n(w),n=e(51),t=e.n(n),c=e(6),j=e(8),_=e(9),b=e(16),C=["children","startIcon","endIcon","onClick","disabled","loading"],M=Object(w.keyframes)(o||(o=S()([`
  from {
    transform: rotate(0deg);
  }
  to {
    transform: rotate(359deg);
  }
`]))),U=O.a.div(a||(a=S()([`
  animation: `,` 2s infinite linear;
  will-change: transform;
`])),M),F=O()(_.Flex)(p||(p=S()([`
  background: transparent;
  border: none;

  &[aria-disabled='true'] {
    pointer-events: none;
    svg path {
      fill: `,`;
    }
  }

  svg {
    display: flex;
    font-size: `,`rem;
  }

  svg path {
    fill: `,`;
  }

  `,`
`])),function(z){return z.theme.colors.neutral600},.625,function(z){return z.theme.colors.primary600},b.a),A=g.a.forwardRef(function(z,q){var K=z.children,ee=z.startIcon,ne=z.endIcon,re=z.onClick,J=z.disabled,Z=z.loading,se=h()(z,C),ce=re&&!J?re:void 0,te=J||Z;return g.a.createElement(F,d()({ref:q,"aria-disabled":te,onClick:ce,as:"button",type:"button"},se),(ee||Z)&&g.a.createElement(c.Box,{as:"span",paddingRight:2,"aria-hidden":!0},Z?g.a.createElement(U,null,g.a.createElement(t.a,null)):ee),g.a.createElement(j.Typography,{variant:"pi",textColor:te?"neutral600":"primary600"},K),ne&&g.a.createElement(c.Box,{as:"span",paddingLeft:2,"aria-hidden":!0},ne))});A.displayName="TextButton",A.defaultProps={disabled:!1,startIcon:void 0,endIcon:void 0,loading:!1,onClick:void 0},A.propTypes={children:x.a.node.isRequired,disabled:x.a.bool,endIcon:x.a.element,loading:x.a.bool,onClick:x.a.func,startIcon:x.a.element}},,,,,,,,,,,,,,,,function(r,s,e){"use strict";e.r(s),e.d(s,"KeyboardNavigable",function(){return w});var o=e(5),a=e.n(o),p=e(30),l=e.n(p),d=e(4),f=e.n(d),h=e(1),m=e.n(h),S=e(0),i=e.n(S),g=e(6),y=e(11),x=["tagName","attributeName"],w=function(O){var n=O.tagName,t=O.attributeName,c=f()(O,x),j=function(){var b=document.activeElement;return n?b.tagName.toLowerCase()===n:b.hasAttribute(t)},_=function(b){return n?b.querySelectorAll(n):b.querySelectorAll("[".concat(t,"]"))};return m.a.createElement(g.Box,a()({onKeyDown:function(b){switch(b.key){case y.a.RIGHT:case y.a.DOWN:if(j()){b.preventDefault();var C=document.activeElement,M=l()(_(b.currentTarget)),U=M.findIndex(function(K){return K===C});M[U+1<M.length?U+1:0].focus()}break;case y.a.LEFT:case y.a.UP:if(j()){b.preventDefault();var F=document.activeElement,A=l()(_(b.currentTarget)),z=A.findIndex(function(K){return K===F});A[z-1>-1?z-1:A.length-1].focus()}break;case y.a.HOME:j()&&(b.preventDefault(),_(b.currentTarget).item(0).focus());break;case y.a.END:if(j()){b.preventDefault();var q=_(b.currentTarget);q.item(q.length-1).focus()}}}},c))};w.defaultProps={attributeName:void 0,tagName:void 0},w.propTypes={attributeName:i.a.string,tagName:i.a.string}},,,,,,,,,,,,,,,,,,,function(r,s,e){"use strict";e.r(s),e.d(s,"AccordionTypography",function(){return j}),e.d(s,"Accordion",function(){return b}),e.d(s,"AccordionContent",function(){return z}),e.d(s,"AccordionToggle",function(){return le}),e.d(s,"AccordionGroup",function(){return de});var o,a,p=e(28),l=e.n(p),d=e(3),f=e.n(d),h=e(1),m=e.n(h),S=e(0),i=e.n(S),g=e(2),y=e.n(g),x=e(8),w=Object(h.createContext)(),O=function(){return Object(h.useContext)(w)},n=e(14),t=e(6),c=e(9),j=y()(x.Typography)(o||(o=f()([""]))),_=y()(t.Box)(a||(a=f()([`
  border: `,`;

  &:hover:not([aria-disabled='true']) {
    border: 1px solid `,`;

    `,` {
      color: `,`;
    }

    `,` {
      color: `,`;
    }

    & > `,` {
      background: `,`;
    }

    [data-strapi-dropdown='true'] {
      background: `,`;
    }
  }
`])),function(u){var T=u.theme,P=u.expanded,W=u.variant,B=u.disabled;return u.error?"1px solid ".concat(T.colors.danger600," !important"):"1px solid ".concat(B?T.colors.neutral150:P?T.colors.primary600:W==="primary"?T.colors.neutral0:T.colors.neutral100)},function(u){return u.theme.colors.primary600},j,function(u){var T=u.theme;return u.expanded?void 0:T.colors.primary700},x.Typography,function(u){var T=u.theme;return u.expanded?void 0:T.colors.primary600},c.Flex,function(u){return u.theme.colors.primary100},function(u){return u.theme.colors.primary200}),b=function(u){var T=u.children,P=u.expanded,W=u.id,B=u.size,H=u.variant,G=u.disabled,N=u.error,L=u.hasErrorMessage,oe=u.onToggle,ae=u.toggle,D=Object(n.a)("accordion",W);return m.a.createElement(w.Provider,{value:{expanded:P,onToggle:oe,toggle:ae,id:D,size:B,variant:H,disabled:G}},m.a.createElement(_,{"data-strapi-expanded":P,disabled:G,"aria-disabled":G,expanded:P,hasRadius:!0,variant:H,error:N},T),N&&L&&m.a.createElement(t.Box,{paddingTop:1},m.a.createElement(x.Typography,{variant:"pi",textColor:"danger600"},N)))};b.defaultProps=l()({disabled:!1,error:void 0,expanded:!1,hasErrorMessage:!0,id:void 0,toggle:!1,size:"M",variant:"primary",onToggle:void 0},"toggle",void 0),b.propTypes={children:i.a.node.isRequired,disabled:i.a.bool,error:i.a.string,expanded:i.a.bool,hasErrorMessage:i.a.bool,id:i.a.string,onToggle:i.a.func,size:i.a.oneOf(["S","M"]),toggle:i.a.func,variant:i.a.oneOf(["primary","secondary"])};var C=e(5),M=e.n(C),U=e(4),F=e.n(U),A=["children"],z=function(u){var T=u.children,P=F()(u,A),W=O(),B=W.expanded,H=W.id;if(!B)return null;var G="accordion-content-".concat(H),N="accordion-label-".concat(H),L="accordion-desc-".concat(H);return m.a.createElement(t.Box,M()({role:"region",id:G,"aria-labelledby":N,"aria-describedby":L},P),T)};z.propTypes={children:i.a.node.isRequired};var q,K,ee=e(34),ne=e.n(ee),re=e(64),J=e(23),Z=e(41),se=["title","description","as","togglePosition","action"],ce=y()(re.TextButton)(q||(q=f()([`
  text-align: left;

  // necessary to make the ellipsis prop work on the title
  > span {
    max-width: 100%;
  }

  svg {
    width: `,`rem;
    height: `,`rem;

    path {
      fill: `,`;
    }
  }
`])),.875,.875,function(u){var T=u.theme;return u.expanded?T.colors.primary600:T.colors.neutral500}),te=y()(c.Flex)(K||(K=f()([`
  min-height: `,`;
  border-radius: `,`;

  &:hover {
    svg {
      path {
        fill: `,`;
      }
    }
  }
`])),function(u){var T=u.theme,P=u.size;return T.sizes.accordions[P]},function(u){var T=u.theme;return u.expanded?"".concat(T.borderRadius," ").concat(T.borderRadius," 0 0"):T.borderRadius},function(u){return u.theme.colors.primary600}),le=function(u){var T=u.title,P=u.description,W=u.as,B=u.togglePosition,H=u.action,G=F()(u,se),N=Object(h.useRef)(null),L=O(),oe=L.onToggle,ae=L.toggle,D=L.expanded,fe=L.id,V=L.size,Ee=L.variant,$=L.disabled,Se="accordion-content-".concat(fe),he="accordion-label-".concat(fe),je="accordion-desc-".concat(fe),ie=V==="M"?6:4,be=V==="M"?ie:ie-2,Pe=function(Y){var ke=Y.expanded,Re=Y.disabled,Me=Y.variant;return ke?"primary100":Re?"neutral150":Me==="primary"?"neutral0":"neutral100"}({expanded:D,disabled:$,variant:Ee}),ze={as:W,fontWeight:V==="S"?"bold":void 0,id:he,textColor:D?"primary600":"neutral700",ellipsis:!0,variant:V==="M"?"delta":void 0},Ce=D?"primary600":"neutral600",_e=D?"primary200":"neutral200",ve="".concat(V==="M"?2:1.5,"rem"),xe=m.a.createElement(c.Flex,{justifyContent:"center",borderRadius:"50%",height:ve,width:ve,transform:D?"rotate(180deg)":void 0,"data-strapi-dropdown":!0,disabled:$,"aria-hidden":!0,as:"span",background:_e,cursor:$?"not-allowed":"pointer",onClick:function(){var Y;return N==null||(Y=N.current)===null||Y===void 0?void 0:Y.click()},shrink:0},m.a.createElement(Z.Icon,{as:ne.a,width:"".concat(V==="M"?11/16:.5,"rem"),color:D?"primary600":"neutral600"}));return m.a.createElement(te,{paddingBottom:be,paddingLeft:ie,paddingRight:ie,paddingTop:be,background:Pe,expanded:D,size:V,justifyContent:"space-between",cursor:$?"not-allowed":""},m.a.createElement(J.Stack,{horizontal:!0,spacing:3,flex:1,maxWidth:"100%"},B==="left"&&xe,m.a.createElement(ce,M()({ref:N,onClick:function(){$||(ae&&!oe?(X.warn('Deprecation warning: Usage of "toggle" prop in Accordion component is deprecated. This is discouraged and will be removed in the next major release. Please use "onToggle" instead'),ae()):oe())},"aria-disabled":$,"aria-expanded":D,"aria-controls":Se,"aria-labelledby":he,"data-strapi-accordion-toggle":!0,expanded:D,type:"button",flex:1,minWidth:0},G),m.a.createElement(m.a.Fragment,null,m.a.createElement(j,ze,T),P&&m.a.createElement(x.Typography,{as:"p",id:je,textColor:Ce},P))),B==="right"&&m.a.createElement(J.Stack,{horizontal:!0,spacing:3},xe,H),B==="left"&&H))};le.defaultProps={action:void 0,as:"span",description:void 0,variant:"primary",togglePosition:"right"},le.propTypes={action:i.a.node,as:i.a.string,description:i.a.string,title:i.a.string.isRequired,togglePosition:i.a.oneOf(["right","left"]),variant:i.a.oneOf(["primary","secondary"])};var pe,me,ge,ye=e(80),we=y()(t.Box)(pe||(pe=f()([`
  border-bottom: 1px solid `,`;
  border-right: 1px solid `,`;
  border-left: 1px solid `,`;
  border-radius: 0 0 `," ",`;
`])),function(u){return u.theme.colors.neutral200},function(u){return u.theme.colors.neutral200},function(u){return u.theme.colors.neutral200},function(u){return u.theme.borderRadius},function(u){return u.theme.borderRadius}),Oe=y()(t.Box)(me||(me=f()([`
  & > * {
    & > * {
      border-radius: unset;
    }
  }

  & > * {
    border-radius: unset;
    border-right: 1px solid `,`;
    border-left: 1px solid `,`;
    border-bottom: 1px solid `,`;
  }

  & > *:first-of-type {
    border-top: 1px solid `,`;
    border-radius: `," ",` 0 0;
    & > * {
      border-radius: `," ",` 0 0;
    }

    &:hover {
      border-top: 1px solid `,`;
    }
  }

  & [data-strapi-expanded='true'] {
    border: 1px solid `,`;
  }

  `,`
`])),function(u){return u.theme.colors.neutral200},function(u){return u.theme.colors.neutral200},function(u){return u.theme.colors.neutral200},function(u){return u.theme.colors.neutral200},function(u){return u.theme.borderRadius},function(u){return u.theme.borderRadius},function(u){return u.theme.borderRadius},function(u){return u.theme.borderRadius},function(u){return u.theme.colors.primary600},function(u){return u.theme.colors.primary600},function(u){var T=u.theme,P=u.footer;return`
    &:not(`.concat(P,`) {
      & > *:last-of-type {
        border-radius: 0 0 `).concat(T.borderRadius," ").concat(T.borderRadius,`;
      }
    }
  `)}),Te=y()(t.Box)(ge||(ge=f()([`
  svg path {
    fill: `,`;
  }
`])),function(u){return u.theme.colors.neutral500}),de=function(u){var T=u.children,P=u.footer,W=u.label,B=u.labelAction,H=u.error,G=h.Children.toArray(T).map(function(N){return Object(h.cloneElement)(N,{hasErrorMessage:!1})});return m.a.createElement(ye.KeyboardNavigable,{attributeName:"data-strapi-accordion-toggle"},W&&m.a.createElement(c.Flex,{paddingBottom:1},m.a.createElement(x.Typography,{variant:"pi",as:"label",textColor:"neutral800",fontWeight:"bold"},W),B&&m.a.createElement(Te,{paddingLeft:1},B)),m.a.createElement(Oe,{footer:P},G),P&&m.a.createElement(we,null,P),H&&m.a.createElement(t.Box,{paddingTop:1},m.a.createElement(x.Typography,{variant:"pi",textColor:"danger600"},H)))};de.defaultProps={footer:null,error:void 0,label:null,labelAction:void 0},de.propTypes={children:i.a.node.isRequired,error:i.a.string,footer:i.a.node,label:i.a.string,labelAction:i.a.node}}])})},98599:function(Q,ue,k){(function(X,I){Q.exports=I(k(32735))})(this,function(X){return function(I){var R={};function E(v){if(R[v])return R[v].exports;var r=R[v]={i:v,l:!1,exports:{}};return I[v].call(r.exports,r,r.exports,E),r.l=!0,r.exports}return E.m=I,E.c=R,E.d=function(v,r,s){E.o(v,r)||Object.defineProperty(v,r,{enumerable:!0,get:s})},E.r=function(v){typeof Symbol!="undefined"&&Symbol.toStringTag&&Object.defineProperty(v,Symbol.toStringTag,{value:"Module"}),Object.defineProperty(v,"__esModule",{value:!0})},E.t=function(v,r){if(1&r&&(v=E(v)),8&r||4&r&&typeof v=="object"&&v&&v.__esModule)return v;var s=Object.create(null);if(E.r(s),Object.defineProperty(s,"default",{enumerable:!0,value:v}),2&r&&typeof v!="string")for(var e in v)E.d(s,e,function(o){return v[o]}.bind(null,e));return s},E.n=function(v){var r=v&&v.__esModule?function(){return v.default}:function(){return v};return E.d(r,"a",r),r},E.o=function(v,r){return Object.prototype.hasOwnProperty.call(v,r)},E.p="",E(E.s=5)}({0:function(I,R){I.exports=X},5:function(I,R,E){"use strict";E.r(R);var v=E(0);function r(){return(r=Object.assign||function(s){for(var e=1;e<arguments.length;e++){var o=arguments[e];for(var a in o)Object.prototype.hasOwnProperty.call(o,a)&&(s[a]=o[a])}return s}).apply(this,arguments)}R.default=function(s){return v.createElement("svg",r({width:"1em",height:"1em",viewBox:"0 0 24 24",fill:"none",xmlns:"http://www.w3.org/2000/svg"},s),v.createElement("path",{d:"M24 13.3a.2.2 0 01-.2.2H5.74l8.239 8.239a.2.2 0 010 .282L12.14 23.86a.2.2 0 01-.282 0L.14 12.14a.2.2 0 010-.282L11.86.14a.2.2 0 01.282 0L13.98 1.98a.2.2 0 010 .282L5.74 10.5H23.8c.11 0 .2.09.2.2v2.6z",fill:"#212134"}))}}})})},37213:(Q,ue,k)=>{var X=k(26460),I=k(88904),R=k(52920),E=k(77236);function v(r,s){var e=E(r)?X:R;return e(r,I(s,3))}Q.exports=v}}]);
